hedera/settings.py

```diff
@@ -8,17 +8,20 @@
 from .aws import get_ecs_task_ips
 
 
+IS_LTI = (os.environ.get("IS_LTI") == "1")
+
 # Initialize Sentry for Error Tracking (see also: https://docs.sentry.io/)
-sentry_sdk.init(
-    dsn=os.environ.get("SENTRY_DSN"),
-    debug=os.environ.get("SENTRY_DEBUG") == "1",
-    environment=os.environ.get("SENTRY_ENVIRONMENT"),
-    integrations=[DjangoIntegration(), RqIntegration()],
-    # Enables tracing for sentry "Events V2"
-    # https://github.com/getsentry/zeus/blob/764df526f47d9387a03b5afcdf3ec0758ae38ac2/zeus/config.py#L380
-    traces_sample_rate=1.0,
-    traceparent_v2=True,
-)
+if not IS_LTI:
```

Comment:
@dodget @arthurian why is Sentry being disabled here?

Reply:
There was some concern over exactly what information sentry is collecting from users, and how it may violate student's FERPA rights. We thought it would be the simpler route to disable it in the LTI, until we have time to go through the Sentry configurations to make sure we aren't in violation.

Reply:
By default PII should be turned off with latest Sentry SDKs which should satisfy FERPA.  We use Sentry in software that we've built in FERPA environments.  But by all means, review this https://docs.sentry.io/data-management/sensitive-data/ and decide for yourself.

Reply:
@dodget what do you think about turning this back on given the default settings for Sentry and PII?